relay-dynamic-config/src/defaults.rs

```diff
@@ -67,7 +67,28 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
             category: DataCategory::Span,
             mri: "d:spans/exclusive_time_light@millisecond".into(),
             field: Some("span.exclusive_time".into()),
-            condition: span_op_conditions,
+            condition: Some(span_op_conditions.clone()),
+            tags: Default::default(),
+        },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/http.response_content_length@none".into(),
+            field: Some("span.data.http\\.response_content_length".into()),
```

Comment:
We have to take this value directly from `data`, because `sentry_tags` is strictly typed to strings.